packages/react/src/components/DateTimePicker/DateTimePickerV2.test.jsx

```diff
@@ -0,0 +1,507 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { render, fireEvent, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import '@testing-library/jest-dom/extend-expect';
+import dayjs from '../../utils/dayjs';
+import {
+  INTERVAL_VALUES,
+  RELATIVE_VALUES,
+  PRESET_VALUES,
+  PICKER_KINDS,
+} from '../../constants/DateConstants';
+
+import DateTimePicker from './DateTimePickerV2';
+import { defaultAbsoluteValue, defaultRelativeValue } from './DateTimePickerV2.story';
+
+const defaultPresets = [
+  ...PRESET_VALUES,
+  {
+    label: 'Last 70 minutes',
+    offset: 70,
+  },
+];
+
+const dateTimePickerProps = {
+  onCancel: jest.fn(),
+  onApply: jest.fn(),
+};
+
+const i18n = {
+  presetLabels: ['Last 30 minutes', 'Missed in translation'],
+  intervalLabels: ['minutes', 'Missed in translation'],
+  relativeLabels: ['Missed in translation'],
+};
+
+describe('DateTimePicker', () => {
+  afterEach(() => {
+    console.error.mockClear();
+  });
+
+  beforeAll(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  afterAll(() => {
+    console.error.mockRestore();
+  });
+
+  it('should not blow up if correct object is passed as default value', () => {
+    mount(
+      <DateTimePicker
```

Comment:
Can we add a test for the testids themselves? As part of the new testing contract we are ensuring that these selectors will always be available.

Reply:
sure, I'm adding for the flyout (other testids are covered it seems under the other testcases